modules/dreamview/backend/sim_control/sim_control.cc

```diff
@@ -66,6 +66,13 @@ SimControl::SimControl(const MapService* map_service)
 
 void SimControl::Init(bool set_start_point, double start_velocity,
                       double start_acceleration) {
+  if (set_start_point && !FLAGS_use_navigation_mode) {
+    start_point_set_ = InitStartPoint(start_velocity, start_acceleration);
+  }
+  if (inited_) {
```

Comment:
There could be a better separation about exactly one time setup init (callback, timer, etc.) and start point init.